geniza/templates/base.html

```diff
@@ -15,6 +15,21 @@
         <meta name="msapplication-TileColor" content="#603cba">
         <meta name="theme-color" content="#6b59c1">
         <meta name="description" content="{% block meta_description %}{% endblock meta_description %}">
+
+        {# open graph metadata #}
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="{% translate "Princeton Geniza Project" %}"/>
+        {# twitter medatada #}
+        <meta name="twitter:domain" content="{{ request.get_host }}" />
+        <meta name="twitter:site" content="@GenizaLab" />
+
```

Comment:
Can we add default page title/description here for all templates? Imagining something like this:

```suggestion
    <meta property="og:title" content="{% firstof page_title page.title %}" />
    <meta property="og:description" content="{% firstof page_description page.description %}" />
    <meta property="og:url" content="{{ request.build_absolute_uri }}" />
    <meta name="twitter:url" content="{{ request.build_absolute_uri }}" />
```